addons/options/src/preview/index.js

```diff
@@ -7,6 +7,19 @@ export function init() {
   // NOTE nothing to do here
 }
 
+let previewOptions = () => {};
+
+function regExpStringify(exp) {
```

Comment:
maybe it worth to submit a separate PR with this feature to have it before we finish with this PR? @igor-dv is it still actual?